src/components/v5/common/MemberSignatureList/MemberSignatureList.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react';
+
+import { SpinnerLoader } from '~shared/Preloaders';
+import MemberSignature from '../MemberSignature';
+import { MemberSignatureListProps } from './types';
+
+const displayName = 'v5.common.MemberSignatureList';
+
+const MemberSignatureList: FC<MemberSignatureListProps> = ({
+  items,
+  isLoading,
+  title,
+}) => {
+  const shouldShowList = !isLoading && !!items?.length;
+
+  return (
+    <div>
+      <h3 className="text-1 mb-2">{title}</h3>
+      {isLoading && (
+        <div className="flex justify-center">
+          <SpinnerLoader appearance={{ size: 'medium' }} />
+        </div>
+      )}
+      {shouldShowList && (
+        <ul>
+          {items.map(({ isChecked, walletAddress, ...rest }) => (
+            <li key={walletAddress} className="mb-3 last:mb-0">
+              {/* @TODO: Implement signed/not signed state */}
```

Comment:
There won't be any logic here, please remove this TODO. The logic will be added in a place where you will use this component and it won't be needed here.